src/style.css

```diff
@@ -1 +1,70 @@
 /* Styles go here */
+
+.body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgb(0 0 0 / 5%);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-list-item {
+  margin-left: 20px;
+}
+
+.nav-list-item:first-child {
+  margin-left: 0;
+}
+
+.nav-list-item > a {
+  text-decoration: none;
+  position: relative;
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  height: 60px;
+  color: #000;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  white-space: nowrap;
+  transition: color 0.3s;
+}
+
+.nav-list-item > a:hover {
+  color: #00acdc;
+}
+
+.nav-list-item > a.is-active {
+  color: #00acdc;
+}
```

Comment:
let's make it simple by removing redundant specificity increment + DRY – you can use comma to combine such styles with the same content inside